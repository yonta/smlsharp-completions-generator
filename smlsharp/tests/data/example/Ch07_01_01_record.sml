val point = {X = 0.0, Y = 0.0};
{1 = 1.1, 2 = fn x => x + 1, 3 = "SML#"};
(1, 2);
fun f x y = {X = x, Y = y};
fun g x y = (x, y);
