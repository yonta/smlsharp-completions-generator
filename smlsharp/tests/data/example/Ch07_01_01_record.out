val point = {X = 0.0, Y = 0.0} : {X: real, Y: real}

val it = (1.1, fn, "SML#") : real * (int -> int) * string

val it = (1, 2) : int * int

val f = fn : ['a. 'a -> ['b. 'b -> {X: 'a, Y: 'b}]]

val g = fn : ['a. 'a -> ['b. 'b -> 'a * 'b]]
